lib/color.ts

```diff
@@ -1,3 +1,9 @@
+interface Color {
```

Comment:
I think to call this function from Typescript, you might need to export this interface. I’m about 60% sure of that. You could try it by converting the color.test.js Jest test to Typescript.

Reply:
This looks like it works okay. I'll commit the test as ts as well